### ISSUE TITLE
Add DELETE articles functionality to the API

### ISSUE THREAD

**Issue by karvounis** [forem/forem#17039]
### Discussed in https://github.com/forem/forem/discussions/17036

<div type='discussions-op-text'>

<sup>Originally posted by **karvounis** March 28, 2022</sup>
### Issue

I am currently working on an open-source tool that leverages the API and aims to manage the Forem resources outside the browser. However, I have found that it is not possible to delete articles through the [API](https://developers.forem.com/api#tag/articles) but it is possible through the browser.

### Feature request

I would love to have the ability to delete articles through the API. I believe it would be a step towards automating creating/updating/deleting articles from outside the browser.

Related to: https://github.com/forem/forem/discussions/17037</div>

**Comment by flimzy** [comment]
Would a PR be accepted to add this support?

**Comment by johnnyreilly** [comment]
How do you delete through the browser? I can't even work that out!

**Comment by johnnyreilly** [comment]
Oh wait - manage and then delete

**Comment by karvounis** [comment]
Yeah it is not very transparent. I doubt there an API endpoint for that

**Comment by mikeydorje** [comment]
Closing due to age and lack of activity. If this is still something you'd like to see, feel free to open a new issue or submit a new PR — contributions are welcome.
